File: 02_Estructuras de Control/03_Repetitivas/01_For/CSharp/ejemplo2.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ejemplo1For
{
    internal class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("  CONTADOR DE NÚMEROS IMPARES Y POSITIVOS");
            Console.WriteLine();

            Console.Write("Ingrese la cantidad de números (n): ");
            int n = int.Parse(Console.ReadLine());

            int contador = 0;

            for (int i = 1; i <= n; i++)
            {
                Console.Write($"Ingrese el número {i}: ");
                int numero = int.Parse(Console.ReadLine());

                // Verificar si es positivo y impar
                if (numero > 0 && numero % 2 != 0)
                {
                    contador++;
                }
            }

            Console.WriteLine();
            Console.WriteLine("==================================");
            Console.WriteLine($"RESULTADO: {contador} número(s) son impares y positivos.");
            Console.WriteLine("==================================");
            Console.ReadKey();
        }
    }
}

File: 02_Estructuras de Control/03_Repetitivas/01_For/Python/ejemplo2.py
print("  CONTADOR DE NÚMEROS IMPARES Y POSITIVOS")
print()

n = int(input("Ingrese la cantidad de números (n): "))

contador = 0

for i in range(1, n + 1):
    numero = int(input(f"Ingrese el número {i}: "))
    
    # Verificar si es positivo y impar
    if numero > 0 and numero % 2 != 0:
        contador += 1

print()
print("==================================")
print(f"RESULTADO: {contador} número(s) son impares y positivos.")
print("==================================")